frost-coordinator/src/coordinator.rs

```diff
@@ -469,51 +469,131 @@ where
 
 #[cfg(test)]
 mod test {
+    use crate::{
+        coordinator::{Command, Coordinator},
+        DEVNET_COORDINATOR_ID,
+    };
```

Comment:
Nit: When test modules are in the same file I prefer to import super::*. It saves a lot of import lines, and since everything is declared in the same file it does not hurt readability.
```suggestion
    use super::*;
```

Reply:
Done.  I had to import DEVNET_COORDINATOR_ID separately.